src/chart/treemap/TreemapSeries.ts

```diff
@@ -54,7 +55,7 @@ interface BreadcrumbItemStyleOption extends ItemStyleOption {
 }
 
 interface TreemapSeriesLabelOption extends SeriesLabelOption {
-    ellipsis?: boolean
+    ellipsis?: TextStyleProps['ellipsis']
```

Comment:
This line can be removed since it's now `string` and is the same as the type in its extended `LabelOption`. So `TextStyleProps` doesn't need anymore accordingly.

Reply:
That's right.